lib/iris/tests/unit/cube/test_CubeAttrsDict.py

```diff
@@ -14,7 +14,7 @@
 from iris.fileformats.netcdf.saver import _CF_GLOBAL_ATTRS
 
 
-@pytest.fixture
+@pytest.fixture()
```

Comment:
The brackets are unnecessary ([based on the docs](https://docs.pytest.org/en/stable/explanation/fixtures.html)), but I don't know if they can actually cause problems.